src/github/client/github-queries.ts

```diff
@@ -469,15 +470,16 @@ export const getDeploymentsQueryWithStatuses = `query ($owner: String!, $repo: S
               createdAt
               updatedAt
               state
-							logUrl
+              logUrl
+              environmentUrl
```

Comment:
This is the only GraphQL changes. Others were my editor auto-fixing spacing of new fields to be consistent with others. I can revert that to clean the diff here if you'd prefer